### PR TITLE
Added PT Translation (OnPlayerObjectMoved) and PT Files fixes

### PR DESCRIPTION
**Pull request by iPollo**

### CODE REVIEW

File: docs/translations/pt/scripting/callbacks/OnPlayerLeaveRaceCheckpoint.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ public OnPlayerLeaveRaceCheckpoint(playerid)
 
 ## Notes
 
-<TipNPCCallbacks />
```
Comment (Southclaws) [inline docs/translations/pt/scripting/callbacks/OnPlayerLeaveRaceCheckpoint.md:31]: We can add a PT template for this.

Comment (Southclaws) [reply docs/translations/pt/scripting/callbacks/OnPlayerLeaveRaceCheckpoint.md:31]: `id` (Indonesian) already has this, simply copy+paste the `id` directory here: https://github.com/openmultiplayer/web/tree/master/frontend/src/components/templates name it `pt` then change the content to:

```ts
export default function TipNpcCallback() {
  return (
    <Admonition type="tip">
      <p>Esta callback também pode ser chamada por um NPC.</p>
    </Admonition>
  );
}
```

And add to `index.ts`:

```ts
import VersionWarnPT from "./pt/version-warning";
import LowercaseNotePT from "./pt/lowercase-note";
import TipNPCCallbacksPT from "./pt/npc-callbacks-tip";
```

**Review by Southclaws — CHANGES_REQUESTED** [review]

**Comment by iPollo** [comment]
Cool, there it is!

**Comment by daddyDOT** [comment]
Maybe you should change those:
`Esta callback também pode ser chamada por um NPC.`
With those, just to generate a "warning box":
```

:::warning

Esta callback também pode ser chamada por um NPC.

:::

```
or if you already done what southclaws told you, you can use this:
`<TipNPCCallbacksPT />`

**Comment by iPollo** [comment]
Yes, I already added the template, I will automatically replace them all tomorrow.

**Pull request merged**